.github/workflows/release_java.yml

```diff
@@ -60,6 +67,16 @@ jobs:
         with:
           version: "23.4"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup linux-aarch_64
+        if: "contains(matrix.classifier, 'linux-aarch_64')"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+          # Setup for cargo
+          echo "[target.aarch64-unknown-linux-gnu]" >> ~/.cargo/config
```

Comment:
May you directly add this config to `bindings/java/.cargo/config`?

Reply:
> May you directly add this config to `bindings/java/.cargo/config`?

This change may break users who want to use musl on aarch. Let's try to use env instead.

Reply:
> use musl

If user uses musl, `target.x86_64-unknown-linux-musl` is used instead? Currently,  `bindings/java/.cargo/config` contains:

```toml
# See also https://github.com/rust-lang/rust/issues/44991

[target.x86_64-unknown-linux-musl]
rustflags = [
    "-C", "target-feature=-crt-static",
]

[target.aarch64-unknown-linux-musl]
rustflags = [
    "-C", "target-feature=-crt-static",
]
```

Reply:
Well, this change LGTM - f6c6ad4ba8ac5d97e9c47aaebfd8553a4f563010